### ISSUE TITLE
Adding some information to navbar

### ISSUE THREAD

**Issue by dragonito** [tolry/mental-note#79]
Is it possible to get information about how much links are added, readed, unreaded to navbar? I think it could be an motivation for reading the old unread Links added in the past.

**Comment by tolry** [comment]
yes, I always wanted to add a statistics section for these kinds of information - you are welcome to start on something like this or just to collect ideas
